index.html

```diff
@@ -4,7 +4,10 @@
   <meta charset="utf-8">
   <meta content="width=device-width, initial-scale=1" name="viewport">
   <title>osdc.github.io</title>
-  <link href="https://fonts.googleapis.com/css?family=Assistant:200|Monoton|Roboto:300" rel="stylesheet">
+  <!-- customized google fonts for main logo -->
+  <link href="https://fonts.googleapis.com/css?family=Monoton&text=OSDC" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Assistant:200|Lato" rel="stylesheet"> 
```

Comment:
Try replacing both of them by:

```
<link rel="preload" href="https://fonts.googleapis.com/css?family=Monoton&text=OSDC" as="style">
<link rel="preload" href="https://fonts.googleapis.com/css?family=Assistant:200|Lato" as="style">
```

Reference: https://www.w3.org/TR/preload/

Reply:
@526avijitgupta this doesn't work even if I use 
```html
<link rel="preload" href="https://fonts.googleapis.com/css?family=Monoton&text=OSDC" as="font" type="font">
```
and as far as fetching crucial resources is concerned we need to load only the monoton font so that the logo doesn't look ugly for few microseconds, but it is already resolved using text=OSDC since it only returns required characters. wdyt?

Reply:
You are right, using preload will not solve the issue (just checked using Chrome's dev tools). In this case then, IMO we should be hiding the text "OSDC" till the font is not fully loaded to prevent the awkward change in font.

Reply:
I think after customizing monoton to return only four characters(OSDC) the problem with the loading logo is resolved. IMO hiding OSDC until the font is loaded can also look awkward. wdyt?

Reply:
Agreed. Let's keep it as it as.